### ISSUE TITLE
Theme update notices may be cut off

### ISSUE THREAD

**Issue by nylen** [ClassicPress/ClassicPress#414]
From https://la-webeuse.com/classicpress-alternative-wordpress/:

<img src="https://la-webeuse.com/wp-content/uploads/2019/04/10-themes.jpg">

The "New version available" notice for Twenty Fifteen, Sixteen, and Seventeen may be cut off in non-English languages.

This code was introduced in #385, and the related CSS (`top: 39px`) needs to be updated to allow the first notice to wrap to multiple lines if needed.

**Comment by bahiirwa** [comment]
1. I find the language above used hard to follow. Is the notice in yellow supposed to overlap the blue? Or the reverse?
2. Does ClassicPress offer a translation for the the new message? 

I am testing this by changing the language pack in general settings of CP Version 1.0.1+migration.20190313

![Screen Shot 2019-06-18 at 16 16 16](https://user-images.githubusercontent.com/7713923/59685328-96cb7d00-91e4-11e9-9031-214f9d6cdaec.png)

**Comment by ginsterbusch** [comment]
my guess is, none of them should overlap each other, but the person who edited the CSS ignored (or was not even awareof) the fact that language strings may vary in length.

So in the original, the first message is supposed to be a short one, and the second one is a longer description of the above "header" message.

cu, w0lf.

**Comment by bahiirwa** [comment]
I have tried fixing but since each language pack has its own height. 

Having `position: absolute` and `position: relative` does not allow for having a dynamic height and positioning. Do we look to having flexbox come into play? That would solve the issue.

Can I use (caniuse.com) suggests Flexbox use:
Global: 94.44% + 3.87% = 98.3%
unprefixed: 94.15% + 2.18% = 96.33%

**Comment by nylen** [comment]
We probably shouldn't introduce flexbox until we make a new major version that intentionally drops support for IE11.

It shouldn't be necessary here though. An alternative approach would be to put all the notices in a container element that has

```css
position: absolute;
left: 0;
right: 0;
top: 0;
bottom: 0;
```

and then have the notices stack normally inside this element with default positioning and `display: block`.

**Comment by nylen** [comment]
>my guess is, none of them should overlap each other, but the person who edited the original CSS in WP ignored (or was not even aware of) the fact that language strings may vary in length.

And this is correct...and that person was me. :man_facepalming:

I know Germans especially like their words long - I think this was one of the last changes to get into v1.0.0 and I forgot to test in another language.

**Comment by Norskes** [comment]
Wrapping is ok
![image](https://user-images.githubusercontent.com/24218656/60302993-92663900-9935-11e9-9339-ab4dfc7d87ad.png)
Note that the markup is messy and a wrapper should be probably applied twise in themes.php (1 for usual php markup itself at str ~260 and 2 in a script template #tmpl-theme-single at str ~400). 
But the real problem is not here. It's in mobile view:

![image](https://user-images.githubusercontent.com/24218656/60303509-dc9bea00-9936-11e9-9bfb-73129742f491.png)

Those containers are too small for the second notice anyway. And if I understand it right, height of containers is controlled by javascript, not CSS. I'll check JS  bit later, but it seems too mon

Do we really need those notices on mobile? I'd offer to hide them. If they are critical, we can display them on :hover at full container height, overlaying the first notice, for example. Something like this (right box is ":hover"):

![image](https://user-images.githubusercontent.com/24218656/60304251-1372ff80-9939-11e9-8482-7f39188c95a8.png)

**Comment by nylen** [comment]
Thanks @Norskes, I am assuming your wrapped screenshots are after making the change I suggested above.

On mobile we can't rely on `:hover`.  I think we could hide the update notice, or another option would be to use a shorter, but less precise message:

> Use "ClassicPress TwentySixteen" instead!

If we go this way we can echo both strings and use CSS breakpoints to show the correct one.
